R/set_seeds.R

```diff
@@ -0,0 +1,82 @@
+##' @title Set Seed for `mlrkeras`
+##' @description `set_seeds`: sets a seed in Random, Python, NumPy and Tensorflow.
+##' Futhermore it disables hash seeds, and can disable GPU and CPU parallesim.
+##' @param seed [`integer`]\cr
+##' A seed to be set on different platforms
+##' @param r_s [`logical`]\cr
+##' Should seed in R be set
+##' @param random_s [`logical`]\cr
+##' Should seed in random be set
+##' @param python_s [`logical`]\cr
+##' Should seed in python/NumPy be set
+##' @param disable_gpu [`logical`]\cr
+##' Should GPU be disabled
+##' @param disable_gpu [`logical`]\cr
+##' Should CPU parallelism be disabled
+##' @rdname set_seeds
+##' @export
+set_seeds = function(seed = 1L,
```

Comment:
rename to `mlr3keras_set_seed` or something that states the purpose more clearly